### DISCUSSION TITLE
CLOG Builder soon to enter beta

### DISCUSSION THREAD

**Discussion by rabbibotton** [rabbibotton/clog#101] [Show and tell]
![crop](https://user-images.githubusercontent.com/11616789/151102972-47cd4a4d-fd8f-4d1d-a848-e04e5531f0d6.png)

All html controls bound
Standard Events
Files are stored as HTML and can import any HTML
Design panels or Websites

Still a little more to go, docs and tutorials but the CLOG Builder is an amazing tool now :)

**Comment by kaeland** [comment]
This looks great!

**Comment by MuhammedZakir** [comment]
Nice one! Looking forward to it! :heart:

**Reply by rabbibotton** [reply]
Too late already 1.0 :)
